### ISSUE TITLE
[buy modal] Align listing purchase / payment success page design to figma

### ISSUE THREAD

**Issue by jabby09** [KlimaDAO/klimadao#1660]
# atmos edit: skip below to my comment to avoid confusion https://github.com/KlimaDAO/klimadao/issues/1660#issuecomment-1874697144

This is related to user flow of buying from a seller listing.

End of transaction payment successful page design in staging using TESTNET
![image](https://github.com/KlimaDAO/klimadao/assets/103535195/36a5740e-7cea-4ebc-9d68-262b96f3edc7)

End of transaction payment successful page design in production using MAINNET
![image](https://github.com/KlimaDAO/klimadao/assets/103535195/e0803a8b-736e-458a-a970-f8e706be3527)

**Comment by 0xMakka** [comment]
@jabby09 do you happen to have a link to the figma files for reference to what this screen should look like?

**Comment by jabby09** [comment]
> @jabby09 do you happen to have a link to the figma files for reference to what this screen should look like?

hey @0xMakka 

I believe the actual figma design for this screen is [here](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=11-59688&mode=design&t=fGPWcbR2vbdVCitf-4).

My recollection of this is when ladytrekker was building this purchase flow, she had questions/issues about the design and for expediency implemented what is in production today (see screenshot below).

We never did go back and update the figma designs.

I can have @theprofessora review all this again (i.e. as part of the CM v2 design changes) but as a short term fix, I just want to make sure the receipt page I saw in TESTNET does not go through to Production. The one we have (below) is good enough for the time being. 

Can you work with what is in production or do you need professor to mock that up for you as a figma file?

![image](https://github.com/KlimaDAO/klimadao/assets/103535195/617036e4-afb4-4381-8f9e-da76044c16d7)

**Comment by 0xMakka** [comment]
> > @jabby09 do you happen to have a link to the figma files for reference to what this screen should look like?
> 
> hey @0xMakka
> 
> I believe the actual figma design for this screen is [here](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=11-59688&mode=design&t=fGPWcbR2vbdVCitf-4).
> 
> My recollection of this is when ladytrekker was building this purchase flow, she had questions/issues about the design and for expediency implemented what is in production today (see screenshot below).
> 
> We never did go back and update the figma designs.
> 
> I can have @theprofessora review all this again (i.e. as part of the CM v2 design changes) but as a short term fix, I just want to make sure the receipt page I saw in staging (using TESTNET) does not go through to Production. The one we have (below) is good enough for the time being.
> 
> It is not clear to me that everything I see in the staging build will automatically get pushed to production. Perhaps that is something you can check with @Atmosfearful. i.e. perhaps that receipt code will not get touched??
> 
> If we do need to change this, can you work with what is in production or do you need professor to mock that up for you as a figma file?
> 
> ![image](https://user-images.githubusercontent.com/103535195/275206299-617036e4-afb4-4381-8f9e-da76044c16d7.png)

Thanks @jabby09, no need for professor to mock up. I just wasn't sure which was the correct screen out of the two. Thanks for the clarification, will get working on this today 👍

**Comment by Atmosfearful** [comment]
I think the discrepancy here is that the 'prod' example @jabby09 gave is a **pool** purchase. I agree we should align the pool purchase UIs to all be identical-- but that is out of scope right now (unrelated to Carbonmark V2)

This is the state that I see after purchase from a listing:
![image](https://github.com/KlimaDAO/klimadao/assets/88635679/3ea763df-e7f2-49b1-86f4-fab4aba91fff)

It is not identical to design, but not too far off.

- I renamed this ticket and marked it as **low priority**
- I created a second ticket for aligning pool purchase pages

**Comment by Atmosfearful** [comment]
you can use these purchase ids @sprrwhwk 
if you don't have test assets to create your own purchase:

https://staging.carbonmark.com/purchases/0x3a3bf8f9ec4f3d8cb9e5613c659bd70f45aeb0cdf72573b25ca2ae6014001336
https://staging.carbonmark.com/purchases/0xa5e7ad3b9e7e8c126746c6809bc5907931de6beb32e551d585b4f4a73f60c6c7

**Comment by jabby09** [comment]
@Atmosfearful @sprrwhwk so this is what flagged this for me in the first place.

When I do a BUY transaction on a Seller listing, this is the end of transaction receipt page that I see (https://staging.carbonmark.com/purchases/0x0d6a302f86f680b33c3c0544dba474dc36864e7622899f442744d095731a7f11) - which does not match 

<img width="772" alt="Screenshot 2023-10-17 at 9 53 43 AM" src="https://github.com/KlimaDAO/klimadao/assets/103535195/4feb26c5-085b-4a26-87bc-a9f40655a7fc">

That being the case, I don't see this as a low priority, since what I am seeing leads the user to believe that they should visit the page again in a few minutes to see another receipt - which is not the case.

I'm perfectly fine with implementing as per the figma https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=11-59688&mode=design&t=81i2TjzM7BgtCx6P-0 and then following up later to align for BUY from pool listing as per #1215

**Comment by sprrwhwk** [comment]
It's a bit hard to follow this with all the edits **AND** later comments

**Comment by Atmosfearful** [comment]
Recap:
- The goal of this ticket is to align the `/purchases/` page to the design shown on figma linked [here ](https://www.figma.com/file/5GewbdVaM1Q06CwHmF7d3F/Projects?type=design&node-id=11-59688&mode=design&t=81i2TjzM7BgtCx6P-0)
  - This /purchases page can only be accessed by buying from a listing with USDC (this is different from buying from a pool, or using Retire Now)
- There seems to be a bug as jabby described. The two retirement links I posted [here ](https://github.com/KlimaDAO/klimadao/issues/1660#issuecomment-1765513343) are not rendering correctly.
- Some additional requirements / expected behavior are described in my comment here https://github.com/KlimaDAO/klimadao/issues/1660#issuecomment-1765513343

**Comment by Atmosfearful** [comment]
> The two retirement links I posted https://github.com/KlimaDAO/klimadao/issues/1660#issuecomment-1765513343are not rendering correctly. I suspect this is only for Mumbai purchases?
Additional note regarding the bug, it's probably because the old subgraph ID expired. The new one is here.
`QmUUnZTeRnfsJsQaTwLeiHmAQ5xvtk2jBW7VeP3AEW5bnv`

**Comment by sprrwhwk** [comment]
> > The two retirement links I posted [#1660 (comment)](https://github.com/KlimaDAO/klimadao/issues/1660#issuecomment-1765513343are) not rendering correctly. I suspect this is only for Mumbai purchases?
> 
> Additional note regarding the bug, it's probably because the old subgraph ID for mumbai expired. ![image](https://private-user-images.githubusercontent.com/88635679/294604995-feb896fd-15ee-490f-8051-f7351ed62fce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ4NzIyNTEsIm5iZiI6MTcwNDg3MTk1MSwicGF0aCI6Ii84ODYzNTY3OS8yOTQ2MDQ5OTUtZmViODk2ZmQtMTVlZS00OTBmLTgwNTEtZjczNTFlZDYyZmNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTEwVDA3MzIzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI4ODM5MTI5MmZiNTc1ZjIxYjM0NDM1Mzg2YzQ4NzYxN2RkMGNmZjA1Y2E4YTNiNzA5MWNhM2M4ZTRjMzE0MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.rVxs1Tsyr3npqyBff2jNls999lWkgkXMOS1hHbzIb_Q)
> 
> The new one is here. `QmUUnZTeRnfsJsQaTwLeiHmAQ5xvtk2jBW7VeP3AEW5bnv`

That sounds like it needs its own bug report. Do we have any other URLs that work?

**Comment by jabby09** [comment]
@sprrwhwk taking this off your list until we do a better discovery / documentation pass
